ceph/rados/core_workflows.py

```diff
@@ -521,8 +522,12 @@ def create_pool(self, pool_name: str, **kwargs) -> bool:
             return False
 
         # Enabling rados application on the pool
-        enable_app_cmd = f"sudo ceph osd pool application enable {pool_name} {kwargs.get('app_name', 'rados')}"
-        self.node.shell([enable_app_cmd])
+        app_name = kwargs.get("app_name", "rados")
+        if app_name:
+            enable_app_cmd = (
+                f"sudo ceph osd pool application enable {pool_name} {app_name}"
+            )
+            self.node.shell([enable_app_cmd])
```

Comment:
This section does not have to be inside a IF block

Reply:
Actually, it looks like the existing code needed no change at all

Reply:
Discussed offline. No change needed, as even None can be a valid app type.